DependencyInjection/LopiPusherExtension.php

```diff
@@ -21,18 +21,33 @@ public function load(array $configs, ContainerBuilder $container)
         $configuration = new Configuration();
         $config = $this->processConfiguration($configuration, $configs);
 
-        $options=array(
-            'cluster'=>$config['cluster'],
-            'debug'=>$config['debug']
-        );
+        if (!empty($config['url'])) {
+            $config['app_id'] = substr(parse_url($config['url'], PHP_URL_PATH), 6);
```

Comment:
Relying on the value of a config parameter in the extension is incompatible with Symfony 3.2 environment variables (cf. symfony/symfony#19681).